src/DonationForms/resources/styles/design-settings/image.scss

```diff
@@ -1,117 +1,158 @@
 .givewp-design-settings--image-style {
     &__background {
-        & > .givewp-layouts-header {
+        .givewp-layouts-header {
             --tint-color: rgba(0, 0, 0, 0.3); /* Note: Color needs to be converted to RGB values. */
             --tint: linear-gradient(90deg, var(--tint-color), var(--tint-color));
             background-color: rgb(30, 140, 190);
             background-image: var(--tint, linear-gradient(transparent, transparent)), var(--givewp-design-settings-background-image);
             background-position: 50%;
             background-size: cover;
+
+            img {
+                display: none;
+            }
+
+            .givewp-layouts {
+                &-headerTitle > h2, &-headerDescription p {
+                    color: var(--givewp-shades-white);
```

Comment:
I don't think we need to solve this now - but what happens if the background color is white?  We'll want a more dynamic adaption of text colors in the near future.